chia/wallet/wallet_node.py

```diff
@@ -689,9 +698,6 @@ async def receive_state_from_peer(
     ) -> bool:
         # Adds the state to the wallet state manager. If the peer is trusted, we do not validate. If the peer is
         # untrusted we do, but we might not add the state, since we need to receive the new_peak message as well.
-
```

Comment:
why remove?

Reply:
this is a lint issue state_manager should never be None after startup
this is since we only initialize it in _start. felt like this should throw and not return without any error.
i changed it to an assert